tests/src/performance/load_test.go

```diff
@@ -8,7 +8,6 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	. "github.com/onsi/gomega/gexec"
-	"github.com/tsenart/vegeta/lib"
```

Comment:
Indeed. I think this was `goimports` removing it because it couldn't resolve it locally on my machine. I've put it back again now.